docs/user/02-10-usage.md

```diff
@@ -3,7 +3,7 @@
 ## Create and Install a Secret
 
 To create a real BTP Manager Secret, follow these steps:
-1. Create ServiceBinding to obtain the access credentials to the ServiceInstance as described in point 2 of the [Setup](https://github.com/SAP/sap-btp-service-operator#setup) section in the SAP BTP service operator documentation.
+1. Create a ServiceBinding to obtain the access credentials to the ServiceInstance as described in the [Setup](https://github.com/SAP/sap-btp-service-operator#setup) section in the SAP BTP service operator documentation (point 2: Obtain the access credentials for the SAP BTP service operator).
```

Comment:
```suggestion
1. Create a ServiceBinding to obtain the access credentials to the ServiceInstance as described in the [Setup](https://github.com/SAP/sap-btp-service-operator#setup) section in the SAP BTP service operator documentation (section: Obtain the access credentials for the SAP BTP service operator).
```

Reply:
This might be confusing as I'm referring to the Setup section already. `Obtain the access credentials for the SAP BTP service operator` is one of the points there. Expressing everything in a more descriptive way results in lengthy sentences with ugly repetitions. Thought adding the actual content might prevent getting completely lost in case of renumbering the original doc. WDYT?